components/atoms/a-product-attribute.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
Moim zdaniem nie ma sensu tworzyć tego componentu. Nie ma tutaj żadnej logiki, której nie dałoby się umieścić po prostu w komponencie parenta, a im więcej komponentów tym trudniej się później w tym odnaleźć.

Reply:
Jest logika, tylko że w mixinie 
`import { ProductAttribute } from '@vue-storefront/core/modules/catalog/components/ProductAttribute';`
i tam jest np odniesienie do `this.attribute` :/ więc musze to wyizolować do osobnego komponentu jak chcę użyć tego mixina.

Reply:
Racja